### PR TITLE
Export fast_beam_search api

### PR DESCRIPTION
**Pull request by pkufool**
Post this just to confirm the api with fangjun @csukuangfj , will add docs and implementations in the following commits.

### CODE REVIEW

File: k2/torch/csrc/torch_api.h [hunk 1]
```diff
@@ -161,6 +161,27 @@ FsaClassPtr GetLattice(torch::Tensor log_softmax_out,
...
 std::vector<std::vector<int32_t>> BestPath(const FsaClassPtr &lattice);
 
+class RnntDecodingStream;
+class RnntDecodingStreams;
+using RnntStreamPtr = std::shared_ptr<RnntDecodingStream>;
+using RnntStreamsPtr = std::shared_ptr<RnntDecodingStreams>;
+
+RnntStreamPtr CreateRnntStream(FsaClassPtr decoding_graph);
+
+RnntStreamsPtr CreateRnntStreams(std::vector<RnntStreamPtr> &source_streams,
```
Comment (csukuangfj) [inline k2/torch/csrc/torch_api.h:171]:
```suggestion
RnntStreamsPtr CreateRnntStreams(const std::vector<RnntStreamPtr> &source_streams,
```

File: k2/torch/csrc/torch_api.cu [hunk 2]
```diff
@@ -69,18 +73,21 @@ FsaClassPtr GetCtcTopo(int32_t max_token, bool modified, torch::Device device) {
   return std::make_shared<FsaClass>(CtcTopo(max_token, modified, device));
 }
 
+FsaClassPtr GetTrivialGraph(int32_t max_token, torch::Device device) {
```
Comment (csukuangfj) [inline k2/torch/csrc/torch_api.cu:76]:
```suggestion
FsaClassPtr GetTrivialGraph(int32_t max_token, torch::Device device /*=torch::kCPU*/) {
```

File: k2/torch/csrc/torch_api.cu [hunk 3]
```diff
@@ -108,4 +115,104 @@ std::vector<std::vector<int32_t>> BestPath(const FsaClassPtr &lattice) {
   return aux_labels_vec;
 }
 
+void ScaleTensorAttribute(FsaClassPtr &fsa, float scale,
+                          const std::string &attribute) {
```
Comment (csukuangfj) [inline k2/torch/csrc/torch_api.cu:119]:
```suggestion
                          const std::string &attribute /*=scores*/) {
```

**Comment by pkufool** [comment]
Can't find pytorch=1.13 for macos, so there are two failed checks.
I am going to merge it.

**Comment by csukuangfj** [comment]
> Can't find pytorch=1.13 for macos, so there are two failed checks. I am going to merge it.

That is fine.

**Pull request merged**